paperqa/settings.py

```diff
@@ -572,6 +573,26 @@ class Settings(BaseSettings):
         frozen=True,
     )
 
+    # imported here to avoid circular ref
+    if TYPE_CHECKING:
+        from .agents.env import EnvironmentState
```

Comment:
Can we move this `TYPE_CHECKING` to the module level, not within the class?

Reply:
yup, my bad